cgra/util.py

```diff
@@ -43,6 +43,7 @@ def create_cgra(width: int, height: int, io_sides: IOSide,
                 use_sim_sram: bool = True,
                 hi_lo_tile_id: bool = True,
                 pass_through_clk: bool = True,
+                tile_layout_option: int = 0, # 0: column-based, 1: row-based
```

Comment:
The original intention is that we may support multiple options in the future. However, I agree for now we can use bool since there is only two options now.

Reply:
If that's the case, use enum to enforce type checking.